apps/ejabberd/include/mongoose_commands.hrl

```diff
@@ -0,0 +1,32 @@
+
+-type command_action() :: [create|read|update|delete]. %% just basic CRUD; sending a mesage is 'create'
```

Comment:
All the types should, IMO, go to the `mongoose_commands` module and be exported from it.